### PR TITLE
fix: dashes group substitution

### PR DESCRIPTION
**Pull request by bogaertg**

### CODE REVIEW

File: slugify.sh [hunk 1]
```diff
@@ -28,12 +28,12 @@ else
...
 
 slug() {
-  output=$(sed -E 's#refs/[^\/]*/##;s/[^a-zA-Z0-9._-]+/-/g;s/-+/-/g;s/^-*//;s/-*$//' <<<"$1")
+  output=$(sed -E 's#refs/[^\/]*/##;s/[^a-zA-Z0-9._-]+/-/g;s/^-*//;s/-*$//' <<<"$1")
   reduce "$output"
 }
 
 slug_url() {
-  output=$(sed -E 's#refs/[^\/]*/##;s/[^a-zA-Z0-9_-]+/-/g;s/-+/-/g;s/^-*//;s/-*$//' <<<"$1")
+  output=$(sed -E 's#refs/[^\/]*/##;s/[^a-zA-Z0-9_-]+/-/g;s/^-*//;s/-*$//' <<<"$1")
```
Comment (rlespinasse) [inline slugify.sh]:
```suggestion
  # 1st : Remove refs prefix
  # 2d : Replace unwanted characters
  # 3d : Remove leading dashes
  # 4d : Remove trailing dashes
  output=$(sed -E 's#refs/[^\/]*/##;s/[^a-zA-Z0-9_]+/-/g;s/^-*//;s/-*$//' <<<"$1")
```

File: slugify.sh [hunk 2]
```diff
@@ -28,12 +28,12 @@ else
 fi
 
 slug() {
-  output=$(sed -E 's#refs/[^\/]*/##;s/[^a-zA-Z0-9._-]+/-/g;s/-+/-/g;s/^-*//;s/-*$//' <<<"$1")
+  output=$(sed -E 's#refs/[^\/]*/##;s/[^a-zA-Z0-9._-]+/-/g;s/^-*//;s/-*$//' <<<"$1")
```
Comment (rlespinasse) [inline slugify.sh]:
```suggestion
  # 1st : Remove refs prefix
  # 2d : Replace unwanted characters
  # 3d : Remove leading dashes
  # 4d : Remove trailing dashes
  output=$(sed -E 's#refs/[^\/]*/##;s/[^a-zA-Z0-9._]+/-/g;s/^-*//;s/-*$//' <<<"$1")
```

File: .github/workflows/slugify-value.yaml [hunk 3]
```diff
@@ -57,10 +57,10 @@ jobs:
       - name: Validate // Slugify key/value
         run: |
           [[ "${{ env.KEY_VALUE_TEST }}" == "refs/pulls/feat/-----Some----Changes_to.be------" ]]
```
Comment (rlespinasse) [inline .github/workflows/slugify-value.yaml:59]: I think we can change the test value to be more useful on each test.
So you can use another value that fit the test case better.

File: slugify.sh (hunk 2 above)
Comment (bogaertg) [reply slugify.sh:31]: docs: add comment for sed

**Comment by rlespinasse** [comment]
Integrated within #14 as co-author

**Pull request closed without merging**